web/app/themes/radcliffe/functions.php

```diff
@@ -122,7 +122,7 @@ function radcliffe_load_style()
             if ('off' !== $google_fonts) {
                 wp_enqueue_style(
                     'radcliffe_googlefonts',
-                    'https://fonts.googleapis.com/css?family=Barlow:300,300i,400,400i,500,500i,600,600i,700,700i|Merriweather:300,300i,400,400i,700,700i,900,900i',
```

Comment:
Nb. the pipe in a Google font query string does not affect any browser and has no adverse effect on accessibility. In fact, this change can impact the successful delivery of the font from Google.

Reply:
Ok. Reverted to original.